entrypoint.py

```diff
@@ -222,6 +237,11 @@ def run_sast(tool, command, env, config_dir, log_file=stdout, run_all=True):
         log.info(f"Skipping {tool}")
         return
 
+    if tool.lower() == "safety" :
+        if (not exists_python_code(os.getenv('PWD'))):
```

Comment:
You can't be sure that the `PWD` environment variable is correct, when you are not in a shell (e.g., bash).

Always use native python functions.

```suggestion
        if (not exists_python_code(os.getcwd())):
```